spec/features/place_index_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature "Places are listed", %Q{
+  As a user
+  I want to see a list of places
+  So that I can decide which place to go
+} do
+
+  scenario 'places are listed in order' do
+    place = Place.create!(name: "Starbucks", place_type: "tea", location: "Boston", description: "description text")
```

Comment:
Because this is an index page, you need to test multiple places (to assure yourself that you're seeing all the places, not just the first one)